nais/nais.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: "nais.io/v1alpha1"
+kind: "Application"
+metadata:
+  name: rekrutteringsbistand-stillingssok-proxy
+  namespace: arbeidsgiver
+  labels:
+    team: arbeidsgiver
+spec:
+  image: {{image}}
+  port: 8300
+  ingresses:
+  {{#each ingress as |url|}}
+      - {{url}}
+  {{/each}}
+  liveness:
+    path: /internal/isAlive
+  readiness:
+    path: /internal/isReady
+  strategy:
+    type: Recreate
+  replicas:
+    min: 1
+    max: 1
+  azure:
+    application:
+      enabled: true
+  elastic:
+    instance: rekrutteringsbistand-stilling
+  # Legg til envFrom: secret
+  env:
+    - name: JAVALIN_PORT
+      value: "8300"
+    - name: ELASTIC_SEARCH_API
+      value: {{elastic-search-api}}
+    - name: PROXY_VERSJON
+      value: "1"
```

Comment:
Hva brukes denne til? Dette er kanskje hentet fra samme config i rekbis-stilling-indekser?
Miljøvariabelen i den appen er spesifikk for reindeksering. Så vil tro vi ikke trenger denne miljøvariabelen her.

Reply:
Bare kopiert ja, så vi fjerner den.